.github/workflows/ci.yaml

```diff
@@ -110,43 +110,43 @@ jobs:
         if: matrix.node-version == '14' && matrix.os == 'ubuntu-20.04'
         run: yarn test
 
-#  release:
-#    name: Release
-#    # https://github.community/t/trigger-job-on-tag-push-only/18076
-#    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
-#    needs: test_push
-#    runs-on: ubuntu-20.04
-#    steps:
-#      - name: Checkout
-#        uses: actions/checkout@v2
-#        with:
-#          persist-credentials: false
-#      - name: Restore target
-#        uses: actions/download-artifact@v2
-#        with:
-#          name: target
-#      - uses: actions/setup-node@v2
-#        with:
-#          node-version: 14
-#
-#      - name: Codeclimate
-#        uses: paambaati/codeclimate-action@v2.7.5
-#        env:
-#          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
-#        with:
-#          coverageLocations: |
-#            ${{github.workspace}}/coverage/*.lcov:lcov
-#      - name: Install deps
-#        run: |
-#          yarn
-#          yarn bootstrap
-#      - name: Multi-semantic-release
-#        env:
-#          GH_TOKEN: ${{ secrets.GH_TOKEN }}
-#          GH_USER: 'qiwibot'
-#          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-#          GIT_AUTHOR_EMAIL: 'opensource@qiwi.com'
-#          GIT_COMMITTER_EMAIL: 'opensource@qiwi.com'
-#          GIT_AUTHOR_NAME: '@qiwibot'
-#          GIT_COMMITTER_NAME: '@qiwibot'
-#        run: yarn release
+  release:
+    name: Release
+    # https://github.community/t/trigger-job-on-tag-push-only/18076
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    needs: test_push
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+      - name: Restore target
+        uses: actions/download-artifact@v2
+        with:
+          name: target
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Codeclimate
```

Comment:
Is CC configured?

Reply:
Yes, I've added repository on codeclimate.com and set secret

Reply:
Yes, I 've added repo and set secret